From 36b733349b8f1358d2de79426189a0c004eebf13 Mon Sep 17 00:00:00 2001
From: P3TERX <p3terx@users.noreply.github.com>
Date: Tue, 1 Sep 2020 02:16:31 +0800
Subject: [PATCH] Disable auto added Want-Digest header

---
 src/HttpRequest.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/HttpRequest.cc b/src/HttpRequest.cc
index b5fb4f86..30fd15d6 100644
--- a/src/HttpRequest.cc
+++ b/src/HttpRequest.cc
@@ -262,7 +262,6 @@ std::string HttpRequest::createRequest()
     }
     if (!wantDigest.empty()) {
       wantDigest.erase(wantDigest.size() - 2);
-      builtinHds.emplace_back("Want-Digest:", wantDigest);
     }
   }
   for (const auto& builtinHd : builtinHds) {
-- 
2.20.1

